app/models/host_initiator_group.rb

```diff
@@ -69,7 +69,7 @@ def self.create_host_initiator_group(ems_id, options = {})
     klass.raw_create_host_initiator_group(ext_management_system, options)
   end
 
-  def self.raw_create_host_initiator_group(_ext_management_system, _options = {})
+  def self.raw_create_host_initiator_group(ext_management_system, options = {})
```

Comment:
This looks like it could stay the way it is currently on master

Reply:
I was concerned that the overriding function wouldn't have the same signature as the base function.
In the overriding function the linter told me to remove the leading underscore, so I adjusted the base function accordingly.

Is it OK to leave this with a leading underscore, but have the child function without one? Or should the child function have an underscore despite what the linter says?

Reply:
Yes as long as the number of arguments is the same then names of the arguments are only local to the specific method and don't matter at all to the overriding method.  You could call them `foo, bar = {}` if you wanted to.  You would only get in trouble if you e.g. only accepted a single argument in the overridden method but callers sent in two args.